lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -33,6 +70,26 @@ <h2 class="message-title hd-5 emphasized">${accomplishment_banner_opening}</h2>
                 <p class="message-copy copy copy-base emphasized">${accomplishment_banner_congrats}</p>
                 <div class="message-actions">
                     <h3 class="sr-only">${_("Take this with you:")}</h3>
+                    % if share_settings:
+                      <%
+                       facebook_share_text = _("I completed the {course_title} course on edX and here is the link to my certificate.").format(course_title=accomplishment_course_title)
```

Comment:
student cannnot change this text but he/she can add its custom update on his/her feed